baseline.yaml

```diff
@@ -960,6 +927,39 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # sastToolRunsOnAllCommits 
 
+  - id: OSPS-QA-07
```

Comment:
qa-07 was introduced earlier, so old qa-05 can not move here.  Is theidea simply to move qa-05 from lvl 2 to lvl 3?  If so, can we just keep it the same id and adjust the level to 3?